splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/BaseRoutes.java

```diff
@@ -0,0 +1,106 @@
+package com.redhat.console.notifications.splunkintegration;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.Processor;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+/* 
+
+1) Adapt code to receive parameter from env variables
+2) Add parameters to application.properties
+3) Refact errorHandlers
+4) Refact ingress
+5) Refact return
+6) Refact success
+
+
+*/
+@ApplicationScoped
+public class BaseRoutes extends ErrorHandler {
+
+  @ConfigProperty(name = "integrations.component.name")
+  String COMPONENT_NAME;
+  
+  // Only accept/listen on these CloudEvent types
+  public String CE_TYPE;
+  
+  // Logger Name for logs using Log EIP
+  public String LOGGER_NAME;
+  
+  // Event incoming kafka brokers
+  @ConfigProperty(name = "kafka.bootstrap.servers")
+  String kafkaBrokers;
+  
+  // Event incoming Kafka topic
+  @ConfigProperty(name = "kafka.ingress.topic")
+  String kafkaIngressTopic;
+  
+  // Event incoming kafka group id
+  @ConfigProperty(name = "kafka.ingress.group.id")
+  String kafkaIngressGroupId;
+  
+  // Event return Kafka topic
+  @ConfigProperty(name = "kafka.return.topic")
+  String kafkaReturnTopic;
+  
+  // Event return kafka group id
+  @ConfigProperty(name = "kafka.return.group.id")
+  String kafkaReturnGroupId;
+  
+  // The return type
+  public static final String RETURN_TYPE = "com.redhat.console.notifications.history";
+
+  @Override
+  public void configure() throws Exception {
+    super.configure();
+
+    CE_TYPE = "com.redhat.console.integration." + COMPONENT_NAME;
```

Comment:
Incorrect type. It should be `"com.redhat.console.notification.toCamel." + COMPONENT_NAME`.
The filer is not working correctly otherwise.

Also can this be defined as a final constant?
There was `CONFIG` defined:
```
private static final Config CONFIG = ConfigProvider.getConfig();
```
that can getValue of a parameter.

Reply:
Nice! I remember I saw this `CONFIG` parameter there, but it was not being used!

Now I could apply it and define other fields as `static final`

Thanks!